src/lib/components/forms/label-input.svelte

```diff
@@ -1,31 +1,54 @@
 <script>
+  export let value = '';
   export let label = '';
   export let id = '';
   export let type = 'text';
   export let placeholder = '';
   export let isTextArea = false;
+  export let required = false;
   export let horizontal = false;
+  export let textGap;
+  export let name = '';
 </script>
 
-<div class="flex flex-{horizontal ? 'row' : 'col'}">
+<div
+  class="flex flex-{horizontal ? 'row justify-between' : 'col'} {$$props.className}"
+  class:align-middle={horizontal && !isTextArea}
+>
   {#if label}
-    <label class="m-1 font-source_code font-bold text-white" for={id}>{label}</label>
+    <div style="width: {textGap}vw;">
```

Comment:
```suggestion
    <div style="width: {textGap ? textGap : '20'}em;">
```
I think vw scales a bit too aggressively here